be/src/runtime/query_context.cpp

```diff
@@ -115,7 +131,9 @@ QueryContext::~QueryContext() {
     }
 
     _exec_env->runtime_query_statistics_mgr()->set_query_finished(print_id(_query_id));
-    LOG_INFO("Query {} deconstructed, {}", print_id(_query_id), mem_tracker_msg);
+
+    _report_query_profile();
```

Comment:
check if enable profile

Reply:
it will be checked in the implementation of _report_query_profile.